index.html

```diff
@@ -5,7 +5,130 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link href="css/index.css" rel="stylesheet" />
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+      rel="stylesheet"
+    />
     <title>Fusion</title>
   </head>
-  <body></body>
+  <body>
+    <!-- avatar -->
+    <!-- <div class="flex pt-44 justify-center">
+          <div class="avatar avatar-xs bg-teal-500">A</div>
+          <div class="avatar avatar-sm bg-teal-500">A</div>
+          <div class="avatar avatar-md bg-teal-500"">A</div>
+          <div class="avatar avatar-lg bg-teal-500">A</div>
+      </div> -->
+    <!-- <div class="flex pt-16 justify-center">
+        <div class="avatar avatar-xs ">
+            <img class="img-round" src="./css/images/img_avatar.png" alt="img" />
+        </div>
+        <div class="avatar avatar-sm "><img class="img-round" src="./css/images/img_avatar.png" alt="img" /></div>
+        <div class="avatar avatar-md"><img class="img-round" src="./css/images/img_avatar.png" alt="img" /></div>
+        <div class="avatar avatar-lg "><img class="img-round" src="./css/images/img_avatar.png" alt="img" /></div>
+    </div> -->
+    <!-- avatar with badge  -->
+    <div class="flex">
+      <div class="pt-16 pl-16">
```

Comment:
Got confused here too, what is pt and pl. 
Your UI looks good , class name conventions are a bit confusing.

Reply:
Pt is padding top, pl is padding left.